ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/port_channel_interfaces.py

```diff
@@ -28,7 +28,7 @@ def port_channel_interfaces(self) -> list | None:
         port_channel_interfaces = []
         port_channel_list = []
         for link in self._underlay_links:
-            if link["type"] != "underlay_l2":
+            if link["type"] != "underlay_l2" or link.get("channel_group_id") is None:
```

Comment:
not sure why we are touching this?

Reply:
Because we now have to handle a single-interface trunk (for the device peering with a wan router with subinterfaces). Since I reused the `underlay_l2` type for those uplinks, we need this.